src/appleseed/renderer/kernel/lighting/bdpt/bdptlightingengine.cpp

```diff
@@ -675,6 +675,10 @@ namespace
             {
             }
 
+            void store_aov_components(const PathVertex& vertex)
```

Comment:
Maybe move after `on_scatter()` since this is typically called last? Please check and confirm. The general idea is to order methods logically, i.e. following some kind of logical progression.